With rust 1.54.0 specifying the version as empty string no longer seems
to be valid. As such, this patch fixes the build with rust 1.54.0 by
specifying the pinned versions from Cargo.lock explicitly.

diff -upr squeekboard-v1.14.0.orig/Cargo.toml.in squeekboard-v1.14.0/Cargo.toml.in
--- squeekboard-v1.14.0.orig/Cargo.toml.in	2021-10-24 10:38:19.438711531 +0200
+++ squeekboard-v1.14.0/Cargo.toml.in	2021-10-24 10:39:18.702889728 +0200
@@ -23,14 +23,14 @@ rustc_less_1_36 = []
 
 # Dependencies which don't change based on build flags
 [dependencies.cairo-sys-rs]
-version = ""
+version = "0.9.2"
 
 [dependencies.glib-sys]
-version = ""
+version = "0.9.1"
 features = ["v2_44"]
 
 [dependencies.gtk-sys]
-version = ""
+version = "0.9.2"
 features = ["v3_22"]
 
 [dependencies]
